zp-relayer/router.ts

```diff
@@ -37,6 +38,7 @@ router.get('/job/:id', wrapErr(endpoints.getJob))
 router.get('/info', wrapErr(endpoints.relayerInfo))
 router.get('/fee', wrapErr(endpoints.getFee))
 router.get('/limits', wrapErr(endpoints.getLimits))
+router.get('/siblings', wrapErr(endpoints.getSiblings))
```

Comment:
makes sense to reflect the new route in the README file

Reply:
Updated